test/integration_test.go

```diff
@@ -107,6 +107,87 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+func TestTerragruntDownloadDir(t *testing.T) {
+	t.Parallel()
```

Comment:
Unfortunately, we can't run this test in parallel because it manipulates environment variables. As in, if this sets an environment variable, it can conflict with other tests running concurrently because the env is set on the test process.